### DISCUSSION TITLE
Save the date: 10 Mai 2022

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#387] [General]
Bonjour à toutes et à tous,

Ca y est enfin !

Grâce à votre super travail, la 4ème édition du référentiel d'écoconception web est enfin prête !

Comme prévu, cette version majeure intègre les nombreuses nouveautés et améliorations:

- référentiel de conformité intégré directement avec les bonnes pratiques ;
- disponible sous deux formats : livre et fiches
- référentiel 100 % ouvert en CC-By-NC-ND

Nous vous convions le 3 mai 2022 de 17h15 à 18h30 pour le lancement auprès du collectif GreenIT.fr et de journalistes.

Il sera également présenté publiquement et officiellement le 10 mai 2022 de 17h15 à 18h45, en ligne.
Nous serions également ravis de pouvoir compter sur votre présence à ce rendez-vous public.

Les liens des conférences en ligne seront communiqués dès que possible.

@AgnesComte
@Alezco
@alpha14
@Altheachevalier
@arnolem
@aureliebaton
@aw73
@bordage
@brpasi
@camcoq-start
@Christophe-Cl
@corentinway
@davidgroult
@devcrafting
@didierfred
@DocRoms
@dupgit
@econuweb
@ekilib666
@EmmanuelDemey
@Evtloglvl
@Ferreole
@florinesueur
@Fred-artwai
@FredMos28
@fzaninotto
@geojs
@jonathandetrierIF
@jpo-softeam
@LaFelixe
@ldesgrange
@LGnord
@mchoraine
@nbordier
@rachelwe
@rlemaire
@tbroyer
@teksaver
@ThomasGiang
@timotheegoguely
@Trankilloman
@Usocrate
@vvatelot
@waku12
@XIAHDEH
@yannkozon

**Comment by EmmanuelDemey** [comment]
C'est un event publique @ACTLEM  ?

**Reply by ACTLEM** [reply]
Merci @EmmanuelDemey, mon message était incomplet. Il y a deux events un plus privé le 3 mai et un public le 10 mai:

```
Nous vous convions le 3 mai 2022 de 17h15 à 18h30 pour le lancement auprès du collectif GreenIT.fr et de journalistes.

Il sera également présenté publiquement et officiellement le 10 mai 2022 de 17h15 à 18h45, en ligne.
Nous serions également ravis de pouvoir compter sur votre présence à ce rendez-vous public.
```

**Reply by ACTLEM** [reply]
Le lien pour le 3 Mai 2022 à 17h15: https://www.gotomeet.me/GreenITfr

**Reply by rlemaire** [reply]
Bonjour,
Est-ce que ce lien fonctionne pour aujourd'hui (10 mai) ?

**Reply by ACTLEM** [reply]
https://www.eventbrite.fr/e/billets-115-bonnes-pratiques-pour-ecoconcevoir-le-web-318861241557

**Reply by rlemaire** [reply]
Pas de lien visio ?

**Reply by ACTLEM** [reply]
Le vrai lien: https://www.google.com/url?q=https://teams.microsoft.com/l/meetup-join/19%253ameeting_OWQzNjZkNTUtY2JiNy00NTZlLWJjOTgtODJkZmFhNDdiNzY2%2540thread.v2/0?context%3D%257b%2522Tid%2522%253a%2522f78f6019-b70c-4b2c-bdab-a9d92656cb98%2522%252c%2522Oid%2522%253a%25223dece8ca-f345-4e52-a7b7-aa1378ff0368%2522%257d&sa=D&source=calendar&usd=2&usg=AOvVaw1ifGsPx1sAxl_0tSli6Ueq

**Reply by rlemaire** [reply]
Merci !

**Reply by EmmanuelDemey** [reply]
Bravo pour hier.

**Comment by Altheachevalier** [comment]
Hello,

3 ou 10 mai ?



De : Thomas LEMAIRE ***@***.***>
Date : lundi, 25 avril 2022 à 18:59
À : cnumr/best-practices ***@***.***>
Cc : Althéa Chevalier ***@***.***>, Mention ***@***.***>
Objet : Re: [cnumr/best-practices] Save the date: 10 Mai 2022 (Discussion #387)

Le lien pour le 3 Mai 2022 à 17h15: https://www.gotomeet.me/GreenITfr

**Reply by ACTLEM** [reply]
3 Mai: Ouvert à la communauté, aux contributeurs ("en avant première")
10 Mai: Tout public

**Comment by rlemaire** [comment]
Je ne pourrais pas être présent le 3 mai (je serais chez un client) mais c'est bon pour le 10 sauf retournement de situation.

**Comment by Christophe-Cl** [comment]
Bonjour Thomas,

Pareil pour moi, je ne serai pas disponible le 03 mais c’est OK pour le 10.

J’espère ne rien manquer de « l’édition du 03 » : si la présentation est vraiment différente, est-ce qu’elle sera enregistrée le 03 afin de la revoir ?




Christophe CLOUZEAU

**Reply by ACTLEM** [reply]
Pour l'enregistrement, je ne sais pas, je me le note.

**Reply by bordage** [reply]
La présentation du 3 sera plus informelle. L'idée est que vous ayez la primeur.  La présentation du 3 sera sur le GotoMeeting du collectif et celle du 10 sur le Teams de Planet Tech Care. Je vais voir s'il est possible d'enregistrer.

**Reply by brpasi** [reply]
Hello. Pour suivi, quid svp de l'enregistrement du 3 ou de celui de ce jour ? 
Pas pu être présent le 3 (et je vais l'être difficilement ce 10 aussi :(), mais je reste preneur comme certains camarades de l'un, l'autre ou les 2 - si écart entre les 2.

Merci et encore bravo !

**Comment by Altheachevalier** [comment]
Hello Thomas,

Je peux jusqu’à 18h00 aujourd’hui,

Bonne journée

De : Thomas LEMAIRE ***@***.***>
Date : samedi, 9 avril 2022 à 14:40
À : cnumr/best-practices ***@***.***>
Cc : Althéa Chevalier ***@***.***>, Mention ***@***.***>
Objet : Re: [cnumr/best-practices] Save the date: 10 Mai 2022 (Discussion #387)

Merci @EmmanuelDemey<https://github.com/EmmanuelDemey>, mon message était incomplet. Il y a deux events un plus privé le 3 mai et un public le 10 mai:

Nous vous convions le 3 mai 2022 de 17h15 à 18h30 pour le lancement auprès du collectif GreenIT.fr et de journalistes.

Il sera également présenté publiquement et officiellement le 10 mai 2022 de 17h15 à 18h45, en ligne.



Nous serions également ravis de pouvoir compter sur votre présence à ce rendez-vous public.

**Comment by Altheachevalier** [comment]
Ca ne fonctionne pas, normal ?

De : Raphaël Lemaire ***@***.***>
Date : mardi, 10 mai 2022 à 17:14
À : cnumr/best-practices ***@***.***>
Cc : Althéa Chevalier ***@***.***>, Mention ***@***.***>
Objet : Re: [cnumr/best-practices] Save the date: 10 Mai 2022 (Discussion #387)

Merci !

**Comment by dNicolle** [comment]
Avec seulement le lien teams ?

Le mar. 10 mai 2022 à 17:19, Altheachevalier ***@***.***> a
écrit :

**Comment by dNicolle** [comment]
Bravo pour ce lancement !

Dominique

**Comment by Christophe-Cl** [comment]
OUI BRAVO par mail car
je n’avais pas accès au tchat sur Teams (pourtant en utilisation « application », curieux)



Bien à vous,

Christophe CLOUZEAU

**Comment by Christophe-Cl** [comment]
Bonjour Thomas

bien noté la différence entre le 03 et le 10 mai !
Je serai présent.

Attention : le 03 mai, le créneau 17h30-18h30 est aussi proposé comme "potentiellement retenable" pour la présentation de la nouvelle association Green-IT. Cf le framadate : https://framadate.org/Jfxujzdo8rU4NJaM


Bien à toi,

Christophe CLOUZEAU
+33 6 89 30 24 27
***@***.*** ***@***.***>
